.github/workflows/tests.yml

```diff
@@ -26,12 +26,9 @@ jobs:
       fail-fast: false
       matrix:
         os: [ubuntu-latest]
-        python-version: [3.6, 3.7, 3.8, 3.9]
-        sphinx: [">=3,<4"]
+        python-version: [3.7, 3.8, 3.9]
```

Comment:
If you are deprecating python 3.6 support, then you need to update setup.cfg

Reply:
and also tox.ini

Reply:
ah right -- roger that. 

Perhaps I won't in the case. This can be done separately if we want to do that as a project.

Reply:
the endo of life is in a few months, so it is fine to do so: https://devguide.python.org/#status-of-python-branches

Reply:
still not sure why

```
tests (ubuntu-latest, 3.8, >=2,<3) Expected — Waiting for status to be reported
```

is expected though.

Reply:
it's part of the branch protection rules

Reply:
@chrisjsewell that end of life chart is pretty great. 👍 I might open an issue to `EOL` python3.6 for `December`

Reply:
> tests (ubuntu-latest, 3.8, >=2,<3) Expected — Waiting for status to be reported

Oh -- thanks I would never have checked there :-)